nix/docker/k8s/cardano-nodes.yaml

```diff
@@ -0,0 +1,190 @@
+#
+# NAMESPACE=cardano
+# kubectl create namespace $NAMESPACE
+#
+
+# GCE Storage Class
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: cardano-standard-rwo
+provisioner: kubernetes.io/gce-pd
```

Comment:
Yes, this is the first cut designed for entry as easy as possible. The full docs are [here](https://tdiesler.gitbook.io/cardano/v/iohk/k8s/cardano-k8s). For now, users can spin up a cluster like this ...

```
$ gcloud container clusters create $CLUSTER_NAME \
  --disk-size=32GB \
  --machine-type=e2-medium \
  --node-locations=$CLUSTER_ZONE \
  --zone=$CLUSTER_ZONE \
  --num-nodes=2
```

To enable portability, which I agree we should in the future, we'd have to provide a ton of docs that address how to setup k8s for various providers and their specifics around storage, firewall, etc.

For now, I'd suggest we stick with GKE until this has matured enough, so that we feel comfortable with adding an additional layer of complexity (i.e. script portability)